docs/pages/5.docsearch.md

```diff
@@ -0,0 +1,81 @@
+---
+title: DocSearch
+description: 'Use Algolia DocSearch in any Nuxt documentation!'
+---
+
+DocSearch provides an easy way for Open Source documentations to have an instant search on their websites!
+
+You probably have seen it in a lot of websites with the **Cmd+K** shortcut!
+
+Websites like [nuxtjs.org](https://nuxtjs.org), [vuejs.org](https://vuejs.org) uses it to provide an instant search to users without any backend!
+
+## Installation
+
+As DocSearch is an additional feature for @nuxtjs/algolia-module, it needs two additional dependencies:
+
+<code-group>
+  <code-block label="Yarn" active>
+
+```bash
+yarn add @docsearch/js @docsearch/css
```

Comment:
Do we need to install these libs separately? Maybe we could install them as a part of the module (they are installed there in the end) and just make the user to switch the boolean of docsearch to true to enable this in his/her app? To me, that provides better DX

Reply:
Installing unnecessary dependencies don't give a better DX to me.

Especially as these packages (`@docsearch/js`) has direct dependency on `preact`, which ultimately leads to installing React into your Vue project.

I think it would be better to tell the user that if he wants to use DocSearch, which is an additional feature from @nuxt-community/algolia-module, he has to install these dependencies.

Reply:
I agree with @Tahul, as long as it's clear they need to install those dependencies separately, it's always better to know what you've actually installed